utils/moreutils/patches/001_disable-manuals.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
```

Comment:
This package won't compile without this patch - the default "all" make rule compiles manpages, and to compile manpages you need some obscure docbook commands (xsltproc?) that I can't seem to find.  There's no other rule in the unmodified makefile that will make all binaries but skip the manpages.